.github/workflows/ci.yaml

```diff
@@ -23,6 +23,9 @@ jobs:
           - name: Test ractor with the `cluster` feature
             package: ractor
             flags: -F cluster
+          - name: Test ractor with the `blanket_serde` feature
```

Comment:
@slawlor 
Not sure how CI works, presumably this works?

Reply:
No you need to add the feature I think comma separated. There's a matrix of tests to run.

Also since this affects cluster we should run it in the docker suite integration tests